tools/shaderc/shaderc_hlsl.cpp

```diff
@@ -663,7 +663,7 @@ namespace bgfx { namespace hlsl
 		uint16_t attrs[bgfx::Attrib::Count];
 		uint16_t size = 0;
 
-		if (_version == 9)
+		if (_version <= 2)
```

Comment:
D3D9 is 3 and below.

Reply:
I reordered the version numbers. Previously we had:
  2. for `s_4_0_level `
  3. for `s_3_0`
  4. for `s_4_0`
  5. for `s_5_0`
 
But now it's (https://github.com/BlockoS/bgfx/blob/master/tools/shaderc/shaderc.cpp#L77):
  2. for `s_3_0`
  3. for `s_4_0_level `
  4. for `s_4_0`
  5. for `s_5_0`

Maybe I should have done something similar to the other targets and use 30, 40, 41, ...

Reply:
> Maybe I should have done something similar to the other targets and use 30, 40, 41, ...

Yeah. 300, 400, 500, etc. Also you could probably kill `s_4_0_level` since those were used for WinRT targets, that are dead anyway.